fe/fe-core/src/main/java/com/starrocks/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -580,6 +580,7 @@ public void write(DataOutput out) throws IOException {
         super.write(out);
         Text.writeString(out, brokerList);
         Text.writeString(out, topic);
+        Text.writeString(out, confluentSchemaRegistryUrl);
```

Comment:
You can't write this URL like this. That will cause incompatibility between new and old versions.

Reply:
I thought be / fe have same version? So how to fix this issue safely?

Reply:
@imay Hi, got time to review this?